### PR TITLE
ETL-3765: Report record count after compaction is done

### PR DESCRIPTION
**Pull request by ydai1124**
@zliu41 Can you review?

### CODE REVIEW

File: gobblin-compaction/src/main/java/gobblin/compaction/mapreduce/MRCompactorJobRunner.java [hunk 1]
```diff
@@ -504,7 +503,7 @@ public int compareTo(MRCompactorJobRunner o) {
     return Double.compare(o.dataset.priority(), this.dataset.priority());
   }
 
-  private List<Path> getCumulativeLateFilePaths(Path lateDataDir) throws FileNotFoundException, IOException {
+  private List<Path> getApplicableFilePaths(Path lateDataDir) throws FileNotFoundException, IOException {
```
Comment (zliu41) [inline gobblin-compaction/src/main/java/gobblin/compaction/mapreduce/MRCompactorJobRunner.java]: Not sure what this method name means. Applicable to what? Please use a more meaningful name and/or add javadoc.
Also if it is not for late data, the parameter should not be called `lateDataDir`

Comment (zliu41) [inline gobblin-compaction/src/main/java/gobblin/compaction/mapreduce/MRCompactorJobRunner.java]: Also `FileNotFoundException` is redundant

**Comment by ydai1124** [comment]
@zliu41 Updated.

**Comment by zliu41** [comment]
LGTM

**Pull request merged**